webui/elm/public/js/determined-shims.js

```diff
@@ -133,6 +136,16 @@ let DetShims = function() {
     });
   }
 
+  function addSetFavicon(app) {
+    app.ports.setFavicon.subscribe((iconPath) => {
+      const linkEl = document.querySelector("link[rel*='shortcut icon']");
+      if (!linkEl) return;
+      linkEl.type = 'image/png';
+      linkEl.rel = 'shortcut icon';
```

Comment:
is this needed if you are matching the linkEl against the same `shortcut icon` attribute value?

Reply:
nope, that's a good point.  don't hold back those observations : )